src/plugins/translation-utils.js

```diff
@@ -0,0 +1,84 @@
+const _ = require('lodash');
+const fs = require('fs');
+const path = require('path');
+
+const languages = [
+  'cs_CZ',
+  'de_DE',
+  'en_ASIA',
+  'en_AU',
+  'en_CA',
+  'en_GB',
+  'en_SG',
+  'en_US',
+  'es_ES',
+  'es_US',
+  'fi_FI',
+  'fr_CA',
+  'fr_FR',
+  'it_IT',
+  'it_LT',
+  'nl_NL',
+  'pl_PL',
+  'pl_PT',
+];
+
+const normalizePath = p => (_.startsWith(p, '.') ? p : `./${p}`);
```

Comment:
```suggestion
const normalizePath = p => (p[0] === '.' ? p : `./${p}`);
```

Reply:
i really prefer startsWith which is more explict (and also p[0] might fail in some cases)

Reply:
I like to get rid of Lodash as much as we can but point taken :)

Reply:
If lodash is available in the project, we should use it. It's like to get a `ferrari`, but doesn't use it because you like your bicycle.

If we want to remove lodash, ok, not a problem, but don't import it and use all ES6+ features :) and use other fallback? :)

Reply:
And startWith do a little more job than `p[0]`

Cf: https://github.com/lodash/lodash/blob/master/startsWith.js

Reply:
Indeed my solution was not adapted, but the best solution may be `p.startsWith('.')` then 😅